### DISCUSSION TITLE
Local tuya

### DISCUSSION THREAD

**Discussion by Merril-r** [community/community#31002] [Other Feature Feedback, Questions, & Ideas]
Hi all,

is the local Tuya repostory from rospogrigio no longer been maintained?

i see there are several bugs posted and unapproved pull requests.

if this is the case is the case there another repository for local tuya that can be recommended?

**Comment by jfullstackdev** [comment]
can you send the link of that ? so we can take a look also,

you can also create an issue to ask question there, maybe others interested will reply too

**Comment by Merril-r** [comment]
https://github.com/rospogrigio/localtuya

I have created and issue on github. There are several people with the same issue. But no repsonse from the repository owneer.

**Comment by airtower-luna** [comment]
The latest commit in that repository is from 11 days ago, the latest release from June. That looks pretty active to me. Keep in mind that people who develop software in their free time might want to do other things than dig through issues and PRs sometimes. :wink:

**Comment by jfullstackdev** [comment]
you might not notice also that it is a forked from another repo, which is the source and the last activity there was 3 years ago

![image](https://user-images.githubusercontent.com/47092464/187033646-e52eb93b-a466-4da1-abfd-9a1912707b5a.png)

![image](https://user-images.githubusercontent.com/47092464/187033666-12d8847b-9b0a-4458-b575-ca6e9b45a42d.png)
